tests/Unit/Domain/CoordinateMaps/Test_Wedge3D.cpp

```diff
@@ -0,0 +1,265 @@
+                                              Direction<3>::lower_eta(),
+                                              0);  // Lower Y wedge
+  const CoordinateMaps::Wedge3D map_lower_xi(inner_r, outer_r,
+                                             Direction<3>::lower_xi(),
+                                             0);  // Lower X wedge
+  const std::array<double, 3> lowest_corner{{-1.0, -1.0, -1.0}};
+  const std::array<double, 3> along_xi{{1.0, -1.0, -1.0}};
+  const std::array<double, 3> along_eta{{-1.0, 1.0, -1.0}};
+  const std::array<double, 3> along_zeta{{-1.0, -1.0, 1.0}};
+
+  // Test that this map's logical axes point along +X, +Y, +Z:
+  CHECK(map_upper_zeta(along_xi)[0] > map_upper_zeta(lowest_corner)[0]);
+  CHECK(map_upper_zeta(along_eta)[1] > map_upper_zeta(lowest_corner)[1]);
+  CHECK(map_upper_zeta(along_zeta)[2] > map_upper_zeta(lowest_corner)[2]);
+
+  // Test that this map's logical axes point along +Z, +X, +Y:
+  CHECK(map_upper_eta(along_xi)[2] > map_upper_eta(lowest_corner)[2]);
+  CHECK(map_upper_eta(along_eta)[0] > map_upper_eta(lowest_corner)[0]);
+  CHECK(map_upper_eta(along_zeta)[1] > map_upper_eta(lowest_corner)[1]);
+
+  // Test that this map's logical axes point along +Y, +Z, +X:
+  CHECK(map_upper_xi(along_xi)[1] > map_upper_xi(lowest_corner)[1]);
+  CHECK(map_upper_xi(along_eta)[2] > map_upper_xi(lowest_corner)[2]);
+  CHECK(map_upper_xi(along_zeta)[0] > map_upper_xi(lowest_corner)[0]);
+
+  // Test that this map's logical axes point along +X, -Y, -Z:
+  CHECK(map_lower_zeta(along_xi)[0] > map_lower_zeta(lowest_corner)[0]);
+  CHECK(map_lower_zeta(along_eta)[1] < map_lower_zeta(lowest_corner)[1]);
+  CHECK(map_lower_zeta(along_zeta)[2] < map_lower_zeta(lowest_corner)[2]);
+
+  // Test that this map's logical axes point along -Z, +X, -Y:
+  CHECK(map_lower_eta(along_xi)[2] < map_lower_eta(lowest_corner)[2]);
+  CHECK(map_lower_eta(along_eta)[0] > map_lower_eta(lowest_corner)[0]);
+  CHECK(map_lower_eta(along_zeta)[1] < map_lower_eta(lowest_corner)[1]);
+
+  // Test that this map's logical axes point along -Y, +Z, -X:
+  CHECK(map_lower_xi(along_xi)[1] < map_lower_xi(lowest_corner)[1]);
+  CHECK(map_lower_xi(along_eta)[2] > map_lower_xi(lowest_corner)[2]);
+  CHECK(map_lower_xi(along_zeta)[0] < map_lower_xi(lowest_corner)[0]);
+}
+
+SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge3D.RandomRadii",
+                  "[Domain][Unit]") {
+  // Set up random number generator:
+  std::random_device rd;
+  std::mt19937 gen(rd());
+  std::uniform_real_distribution<> real_dis(-1, 1);
+  std::uniform_real_distribution<> inner_dis(1, 3);
+  std::uniform_real_distribution<> outer_dis(4, 7);
+
+  // Check that points on the corners of the reference cube map to the correct
+  // corners of the wedge.
+  const std::array<double, 3> inner_corner{{-1.0, -1.0, -1.0}};
+  const std::array<double, 3> outer_corner{{1.0, 1.0, 1.0}};
+  const double random_inner_radius_upper_xi = inner_dis(gen);
```

Comment:
This block and the next few have statements ordered up_xi, up_eta, lo_xi, lo_eta, up_zeta, lo_zeta.  Please either group by axis or up/lo.